megcruzen/browserify_productOrdering
# Product list is now built using database

# Description
Product list is now built using database.

## Type of change

# Testing Instructions for Change Made
1. `git fetch --all`
2. `git checkout fetch-product-data`
3. Run server/grunt
4. Verify that products (currently two) display on the DOM.

# Checklist:
